code/modules/spells/spell_types/lightning.dm

```diff
@@ -81,4 +81,64 @@
 			return
 		var/mob/living/next = pick(possible_targets)
 		if(next)
-			Bolt(current,next,max((bolt_energy-5),5),bounces-1,user)
\ No newline at end of file
+			Bolt(current,next,max((bolt_energy-5),5),bounces-1,user)
+
+/obj/effect/proc_holder/spell/lightningbolt
+	name = "Lightning Bolt"
+	desc = "Fire a high powered lightning bolt at your foes!"
+	school = "evocation"
+	charge_max = 200
+	clothes_req = 1
+	invocation = "UN'LTD P'WAH"
+	invocation_type = "shout"
+	cooldown_min = 30
+	var/projectile_type = /obj/item/projectile/magic/aoe/lightning
```

Comment:
you're _still_ doing something wrong here, given that this and the fireball spell are not both subtypes of a type with this var